File: sdk/python/examples/05_kv_ttl.py
"""05_kv_ttl:TTL / persist。"""
from helpers import client

combee = client()
cell = combee.cells.create()
cell.kv.set("session", "abc", ttl=60)
print("ttl:", cell.kv.ttl("session"))
cell.kv.persist("session")
print("after persist:", cell.kv.ttl("session"))

File: sdk/typescript/examples/05_kv_ttl.mjs
// 05_kv_ttl:TTL / persist。
import { client } from "./00_helpers.mjs";
const combee = client();
const cell = await combee.cells.create();
await cell.kv.set("session", "abc", { ttl: 60 });
console.log("ttl:", await cell.kv.ttl("session"));
await cell.kv.persist("session");
console.log("after persist:", await cell.kv.ttl("session"));
